virttest/env_process.py

```diff
@@ -1241,6 +1241,17 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Get the version of bootloader
+    vm_bootloader_cmd = params.get("vm_bootloader", "")
+    if vm_bootloader_cmd:
+        try:
+            vm_bootloader_ver = a_process.run(
+                vm_bootloader_cmd, shell=True).stdout_text.strip()
+            LOG.debug("vm_bootloader_version: %s", vm_bootloader_ver)
+            version_info["vm_bootloader_ver"] = str(vm_bootloader_ver)
+        except a_process.CmdError:
+            vm_bootloader_ver = "Unkown"
```

Comment:
Even if the version is "Unknown", it can also be saved in the version_info, like qemu/libvirt doing.

Reply:
Thanks, updated